src/platforms/kotlin-multiplatform/common/install/cocoapods.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Cocoapods
```

Comment:
Why do we need this file if the `getting-started-install` section already have the cocoapods setup?

Reply:
I assume users would expect all installation methods to be listed under `Apple Installation Methods` so if only SPM is present there it might not be obvious that it's in getting started especially if someone skips the getting started tutorial or tries searching for `sentry kmp cocoapods installation` through google.

Reply:
For me it's confusing that the [Install step](https://sentry-docs-fwte4vsy1.sentry.dev/platforms/kotlin-multiplatform/#install) already installs the Apple dep via `cocoapods` but just after comes the `Next Steps` with `Apple Installation Methods`, feel like you have to do it again, or additional steps.
We can improve this later if users find it confusing.

Reply:
I agree this is a bit confusing. Maybe a note that says if you've followed the "Getting Started" setup you will have already done this would help.